### ISSUE TITLE
outputClientImpl=grpc-web broken

### ISSUE THREAD

**Issue by ngrok-sarid** [stephenh/ts-proto#174]
Using `1.40.0`:

```shell
[/home/ngrok/ngrok] protoc --plugin=js/node_modules/.bin/protoc-gen-ts_proto --ts_proto_opt=outputClientImpl=grpc-web,env=browser,unrecognizedEnum=false --ts_proto_out=js/src/nlib/gen2 --proto_path=proto -Iproto/thirdparty proto/agent/*.proto
FAILED!unused arguments: expected 2, received 3Error: unused arguments: expected 2, received 3
    at Object.check (/home/ngrok/ngrok/js/node_modules/ts-poet/build/utils.js:6:15)
    at CodeBlock.add (/home/ngrok/ngrok/js/node_modules/ts-poet/build/CodeBlock.js:246:21)
    at createUnaryMethod (/home/ngrok/ngrok/js/node_modules/ts-proto/build/generate-grpc-web.js:178:51)
    at generateGrpcWebImplPromise (/home/ngrok/ngrok/js/node_modules/ts-proto/build/generate-grpc-web.js:151:22)
    at Object.addGrpcWebMisc (/home/ngrok/ngrok/js/node_modules/ts-proto/build/generate-grpc-web.js:109:87)
    at Object.generateFile (/home/ngrok/ngrok/js/node_modules/ts-proto/build/main.js:119:40)
    at /home/ngrok/ngrok/js/node_modules/ts-proto/build/plugin.js:19:29
    at Array.map (<anonymous>)
    at main (/home/ngrok/ngrok/js/node_modules/ts-proto/build/plugin.js:18:37)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)--ts_proto_out: protoc-gen-ts_proto: Plugin failed with status code 1.
```

- Replacing `outputClientImpl=grpc-web` with `outputClientImpl=true` (or just deleting) does not cause a problem.
- Adding `returnObservable` fixes the problem, but now everything is an `Observable`.

I am not sure what to share, so please ask.

**Comment by stephenh** [comment]
Thanks for reporting; I'm looking into this / working on a fix.

**Comment by stephenh** [comment]
This should be fixed in v1.41.0. Let me know how it works, thanks!

**Comment by ngrok-sarid** [comment]
It works. Thank you for fixing this! :smile:
